src/api/response_models/user.py

```diff
@@ -29,3 +29,7 @@ class UserInfoResponse(BaseModel):
 
 class UserWithStatusResponse(UserResponse):
     status: User.Status
+
+
+class UserDetailResponse(UserResponse):
+    shifts: list

```

Comment:
Здесь неплохо бы указать ShiftResponse-модель, но получил ошибку циркулярного импорта, с наскока не смог решить. Возможно, следует пересмотреть порядок всех импортов для Response-моделей.

Reply:
Посмотрю. Напишу.

Reply:
Попробуй вынести импорты внутри модуля в __init__
И здесь будет не просто ShiftResponse. Там же еще есть новые поля, которые нужно также отобразить в схеме Swagger.

Reply:
Привел `src/api/response_models/__init__.py` к виду:
```python
from src.api.response_models.member import MemberResponse
from src.api.response_models.shift import ShiftUserDetailResponse
from src.api.response_models.task import TaskInfoResponse
from src.api.response_models.user import UserResponse, UserInfoResponse
```
В модулях переписал соответствующие импорты.
Продолжаю получать ошибку:
```
ImportError: cannot import name 'UserResponse' from partially initialized module 'src.api.response_models' (most likely due to a circular import) (src\api\response_models\__init__.py)
```
Получается такая схема импортов (если не переносить их в `__init__`):
![image](https://user-images.githubusercontent.com/70967111/209853685-b9f63581-5712-48f6-a5de-a44f0da83776.png)

Reply:
Если файл DTO_models.py превратить в паке и сделать в нём два файла, то эта проблема решится, так как в шифт_дто не будет обращения к респонс модели.

Reply:
Не понимаю, почему здесь упоминаешь `DTO_models`.
С ним как раз-таки проблем нет - там просто описывается промежуточная (между уровнями репозитория и сервиса) модель и в пакете `DTO_models` нет обращений к `response_models`.
Проблема с уровнем представления, а именно с пакетом `response_models`, модули которого импортируют модели друг-друга для составления сложных структур ответа.

Reply:
Самый простой вариант решения такой. 

В файле shift.py импорт перенести к модели. 
```suggestion
from src.api.response_models.member import MemberResponse


class ShiftMembersResponse(BaseModel):
    """Модель для вывода пользователей смены."""

    shift: ShiftResponse
    members: list[MemberResponse]

```

Другие не работают, нужно изменять структуру модуля. Этим лучше заняться когда будут реализованы все модели.